test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -234,17 +240,21 @@ public void OnnxStatic()
         [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
         void TestCommandLine()
         {
+            // Onnxruntime supports Ubuntu 16.04, but not CentOS
+            // Do not execute on CentOS image
             if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 return;
 
             var env = new MLContext();
-            var x = Maml.Main(new[] { @"showschema loader=Text{col=data_0:R4:0-150527} xf=Onnx{InputColumns={data_0} OutputColumns={softmaxout_1} model={squeezenet/00000001/model.onnx} GpuDeviceId=0 FallbackToCpu=+}" });
+            var x = Maml.Main(new[] { @"showschema loader=Text{col=data_0:R4:0-150527} xf=Onnx{InputColumns={data_0} OutputColumns={softmaxout_1} model={squeezenet/00000001/model.onnx} GpuDeviceId=" + Int32.MinValue + " FallbackToCpu=+}" });
```

Comment:
Does `Int32.MinValue` mean something specific?

Reply:
If it is something specific, please define it as a const (in OnnxUtils, OnnxModel, or wherever makes sense) with the proper descriptive name. Magic numbers are confusing.

---
In reply to: [251942412](https://github.com/dotnet/machinelearning/pull/2085#discussion_r251942412) [](ancestors = 251942412)

Reply:
Defined "public const int NullGpuID = int.MinValue" in OnnxTransformer class. OnnxModel and OnnxUtils class was made "internal sealed" in previous round, since we don't want to expose externally.

---
In reply to: [252010420](https://github.com/dotnet/machinelearning/pull/2085#discussion_r252010420) [](ancestors = 252010420,251942412)